README.md

```diff
@@ -6,9 +6,16 @@
 [![Tests](https://github.com/poker-io/pokerio-server/actions/workflows/test.yml/badge.svg)](https://github.com/poker-io/pokerio-server/blob/main/.github/workflows/test.yml)
 [![codecov](https://codecov.io/gh/poker-io/pokerio-server/branch/main/graph/badge.svg?token=4QCZNOWFZJ)](https://codecov.io/gh/poker-io/pokerio-server)
 
-## Building
+## Requirements
```

Comment:
Maybe it is a good time to add information about additional required files to run the serwer, such as serviceAccount.json?

Reply:
The info about this is in line 22, under the `Running` section

Reply:
The info about this is in line 22, under the `Running` section

Reply:
Allright but what about file secrets.ts?

Reply:
Added all the info to README, *thanks* for the comment